libssu/ssufeaturemanager.cpp

```diff
@@ -36,11 +36,6 @@ QStringList SsuFeatureManager::repos(bool rndRepo, int filter)
 {
     QStringList r;
 
-    // @TODO features currently can't be blacklisted, but just ignoring user filter
```

Comment:
This was a bit of a hack, and this is only called when the BoardFilter is enabled so it's no longer needed

Reply:
Best to still leave it checking the board filter